app/Jobs/SendSubscribedNotificationsJob.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+namespace App\Jobs;
+
+use App\Mail\CheckInNotificationMail;
+use App\Models\PhoneNumber;
+use App\Models\Subscription;
+use Illuminate\Bus\Queueable;
+use Illuminate\Contracts\Queue\ShouldBeUnique;
+use Illuminate\Contracts\Queue\ShouldQueue;
+use Illuminate\Foundation\Bus\Dispatchable;
+use Illuminate\Queue\InteractsWithQueue;
+use Illuminate\Queue\SerializesModels;
+use Illuminate\Support\Facades\Log;
+use Illuminate\Support\Facades\Mail;
+
+class SendSubscribedNotificationsJob implements ShouldQueue, ShouldBeUnique
```

Comment:
ShouldBeUnique prevents a ton of emails going out if multiple check ins occur in a short window